model/src/main/kotlin/licenses/ResolvedLicenseInfo.kt

```diff
@@ -112,25 +113,46 @@ data class ResolvedLicenseLocation(
     /**
      * All copyright findings associated to this license location.
      */
-    val copyrights: Set<ResolvedCopyrightFinding>
+    val copyrights: Set<ResolvedCopyright>
+)
+
+/**
+ * A resolved copyright.
+ */
+data class ResolvedCopyright(
+    /**
+     * The resolved copyright statement.
+     */
+    val statement: String,
+
+    /**
+     * The resolved findings for this copyright. The statements in the findings can be different to [statement] if they
+     * were processed by the [CopyrightStatementsProcessor].
+     */
+    val findings: Set<ResolvedCopyrightFinding>,
+
+    /**
+     * True, if this [statement] is contained in the [CopyrightGarbage] used during resolution.
```

Comment:
No, this is set to true only if the statement above is garbage, the original statements are separately marked as garbage in `ResolvedCopyrightFinding` below. I would like to address the topic of handling copyright garbage in a quick call tomorrow, because there are a few other things I'm undecided about.

Reply:
As discussed let's change the code (In some following PR) to apply garbage filtering exclusively on the `raw` statements, for (1) clarity / simplicity and (2) keeping the garbage entries independent of any processing logic.